### DISCUSSION TITLE
Virtual Select Search Box

### DISCUSSION THREAD

**Discussion by yasinkt** [sa-si-dev/virtual-select#183] [Q&A] [answered]
search box value clear after selected item 

How can I do ?

**Comment by sa-si-dev** [comment]
> search box value clear after selected item 
> 
> How can I do ?

As of now, this feature not available.

I will add it in the future release.

As a workaround, you could clear search value by calling click method on `.vscomp-search-clear` element

**Reply by yasinkt** [reply]
'virtual-select' hideing function is there ?

**Reply by sa-si-dev** [reply]
> 'virtual-select' hideing function is there ?

You could use `close()` method

```js
document.querySelector('#sample-select').close();
```

**Comment by sa-si-dev** [comment, marked as answer]
@yasinkt Search value reset on option selection feature available in the release [v1.0.33](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.33)

**Reply by yasinkt** [reply]
how do i use it ?

**Reply by sa-si-dev** [reply]
This is the default behaviour.
- Search options
- Select any option
- Dropbox will close and search value would be reset

**Reply by yasinkt** [reply]
okay it's worked but it has to be like this: search box value clear after closeing virtual-select ? can you do it this way? I would be grateful if you do this for single and multiple selections.

**Reply by yasinwise** [reply]
Is the transaction complete?
